docs/infrastructure/workers/worker-tools-versioning-and-caching.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Worker Tools, Versioning and Caching
+description: Worker Tools 
+position: 50
+---
+
+Worker Tools are a set of docker images used as [execution containers for workers](https://octopus.com/docs/projects/steps/execution-containers-for-workers)] to run deployment processes. Worker Tools include a wide range of software tools with the aim of supporting a majority of deployment scenarios out of the box. This post focuses on how we create these Worker Tool images, version, cache on workers and release them.
+
+## Versioning Worker Tools
+Worker Tool images, follow a semantic versioning (SemVer) approach of `Major.Minor.Patch-Distro` for their tag format. When we release a new version of Worker Tools to the [Worker Tools dockerhub repository](https://hub.docker.com/r/octopusdeploy/worker-tools/tags) we also add the following image tags, distribution (`ubuntu.18.04` or `windows.ltsc2019`), `Major-Distro` and `Major.Minor-Distro`. We recommend using the fully qualified SemVer as patch updates of Worker Tools could result in an updated tool depenedncy introducing a breaking change.
+
+The Worker Tools Dockerfiles use a combination of tools pinned to specifc versions such as CLI tools, Frameworks while other tools pull their latest avaliable release, for Ubuntu these are pulled with apt-get and for Windows chocolatey. You can find the full details of these tools in the docker files for [Windows](https://github.com/OctopusDeploy/WorkerTools/blob/master/windows.ltsc2019/Dockerfile) and [Ubuntu](https://github.com/OctopusDeploy/WorkerTools/blob/master/ubuntu.18.04/Dockerfile) Worker Tools.
+
+The tools pulling their latest releases for Ubuntu include `wget`, `python3-pip`, `groff`, `unzip`, `apt-utils`, `curl`, `software-properties-common`, `jq`, `yq`, `openssh-client`, `rsync`, `git`, `augeas-tools`, `maven`, `gradle`, `Node 14`, `istioctl`, `linkerd `, `umoci`.
+
+For Windows these include `chocolatey` and `dotnet core 3.1.*`.
+
+If you depend on any of these packages we recommend you target the fully qualified SemVer of Worker Tools. Otherwise our additional Worker Tools tags may be suitable for your use case. We version our releases as follows:
```

Comment:
```suggestion
If you depend on any of these packages, we recommend you target the fully qualified SemVer of Worker Tools. Otherwise, our additional Worker Tools tags may be suitable for your use case. We version our releases as follows:
```

Reply:
Maybe `... your steps depend on ...` is a bit clearer than `... you depend on...`?